### DISCUSSION TITLE
disable/enable swipe gesture in SideMenuView

### DISCUSSION THREAD

**Discussion by ghost** [xamarin/XamarinCommunityToolkit#1704] [Q&A] [answered]
Hi everyone, I want to ask if it is possible to disable / enable swipe gesture in SideMenuView in some way, for example in [SfNavigationDrawer ](https://help.syncfusion.com/xamarin/navigation-drawer/swipe-gesture-and-sensitivity)EnableSwipeGesture has this property.

Thanks a lot.

**Comment by bijington** [comment, marked as answer]
@ORRNY66 looking through the code I can see that there is a mechanism to `SetMenuGestureEnabled`. I haven't tested this myself but it would be worth seeing if this will provide what you need.

https://github.com/xamarin/XamarinCommunityToolkit/blob/2c267757be8aa4e6b4ce8371592628950af0572f/src/CommunityToolkit/Xamarin.CommunityToolkit/Views/SideMenuView/SideMenuView.shared.cs#L160

**Reply by ghost** [reply]
Thanks a lot, it helped.
